compiler/crates/relay-lsp/src/server/lsp_state.rs

```diff
@@ -516,8 +523,26 @@ impl<TPerfLogger: PerfLogger + 'static, TSchemaDocumentation: SchemaDocumentatio
             js_server.process_js_source(uri, text);
         }
 
+        let source_location = SourceLocationKey::standalone(uri.path());
```

Comment:
TODO: Use FileClassifier to determine whether the file is a schema (extension) file